mlflow/pytorch/__init__.py

```diff
@@ -248,36 +248,21 @@ def log_model(
 
         import numpy as np
         import torch
-        import mlflow.pytorch
-
-
-        class LinearNNModel(torch.nn.Module):
-            def __init__(self):
-                super().__init__()
-                self.linear = torch.nn.Linear(1, 1)  # One in and one out
-
-            def forward(self, x):
-                y_pred = self.linear(x)
-                return y_pred
-
-
-        def gen_data():
-            # Example linear model modified to use y = 2x
-            # from https://github.com/hunkim/PyTorchZeroToAll
-            # X training data, y labels
-            X = torch.arange(1.0, 25.0).view(-1, 1)
-            y = torch.from_numpy(np.array([x * 2 for x in X])).view(-1, 1)
-            return X, y
```

Comment:
This function was broken with the latest Pytorch libraries. I've switched it out for the following code snippet.
```python
X = torch.arange(1.0, 26.0).reshape(-1, 1)
y = X * 2
```